### PR TITLE
Env param injection

### PR DESCRIPTION
**Pull request by ryan-cahill**
GitLab ticket: https://gitlab.com/architect-io/hub-api/issues/51

### CODE REVIEW

File: src/common/docker-compose/index.ts [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
 import path from 'path';
-import DependencyManager, { DatastoreNode, ServiceNode } from '../../dependency-manager/src';
+import DependencyManager, { DatastoreNode, DependencyNode, ServiceNode } from '../../dependency-manager/src';
 import { ExternalNode } from '../../dependency-manager/src/graph/node/external';
 import { LocalServiceNode } from '../dependency-manager/local-service-node';
 import DockerComposeTemplate from './template';
 
+const inject_params = (environment: { [key: string]: string | number }, data_node: DependencyNode, ) => {
+  const param_prefix = `$ARC_${data_node.name.replace(/[^\w\s]/gi, '_').toUpperCase()}_${data_node.tag.replace(/[^\w\s]/gi, '_').toUpperCase()}`;
+  const host_param_placeholder = `${param_prefix}_HOST`;
```
Comment (tjhiggins) [inline src/common/docker-compose/index.ts]: Does add a backslash to the $ not work as opposed to adding a module?
`\${param_prefix}_HOST`;

Comment (ryan-cahill) [reply src/common/docker-compose/index.ts]: That would escape the entire variable that we're trying to insert into the string, wouldn't it?

Comment (tjhiggins) [reply src/common/docker-compose/index.ts]: What is the output of escapeRegExp?

Comment (ryan-cahill) [reply src/common/docker-compose/index.ts]: Fixed

**Pull request merged**